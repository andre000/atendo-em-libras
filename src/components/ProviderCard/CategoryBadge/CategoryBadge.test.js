import React from 'react'
import { CategoryBadge } from '.'
import { render } from '@testing-library/react'

describe('CategoryBadge', () => {
  it('renders medicine correctly', () => {
    const medicineBadge = render(<CategoryBadge category="Medicina" />)

    expect(medicineBadge).toMatchSnapshot()
  })

  it('renders advocacia as default correctly', () => {
    const lawyerBadge = render(<CategoryBadge category="Advocacia" />)

    expect(lawyerBadge).toMatchSnapshot()
  })

  it('renders pessoa cuidadora correctly', () => {
    const careGiverBadge = render(<CategoryBadge category="Pessoa Cuidadora" />)

    expect(careGiverBadge).toMatchSnapshot()
  })

  it('renders psicologia case correctly', () => {
    const psychology = render(<CategoryBadge category="Psicologia" />)

    expect(psychology).toMatchSnapshot()
  })

  it('renders enfermagem case correctly', () => {
    const nursing = render(<CategoryBadge category="Enfermagem" />)

    expect(nursing).toMatchSnapshot()
  })

  it('renders fisioterapia case correctly', () => {
    const physiotherapy = render(<CategoryBadge category="Fisioterapia" />)

    expect(physiotherapy).toMatchSnapshot()
  })

  it('renders odontologia case correctly', () => {
    const dentistry = render(<CategoryBadge category="Odontologia" />)

    expect(dentistry).toMatchSnapshot()
  })

  it('renders fonoaudiologia case correctly', () => {
    const speechTherapy = render(<CategoryBadge category="Fonoaudiologia" />)

    expect(speechTherapy).toMatchSnapshot()
  })

  it('renders nutrição case correctly', () => {
    const nutrition = render(<CategoryBadge category="Nutrição" />)

    expect(nutrition).toMatchSnapshot()
  })
})
