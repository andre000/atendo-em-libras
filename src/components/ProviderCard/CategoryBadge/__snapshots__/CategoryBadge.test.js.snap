// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryBadge renders advocacia as default correctly 1`] = `
.c0 {
  margin-right: 1em;
  color: #fff;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  max-width: 70%;
  font-size: 1.4em;
  font-weight: 600;
  padding: 0.2em 2em;
  border-radius: 1.5em;
  line-height: 1.8em;
  background-color: #5A8DED;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 1.5em;
}

<div>
  <p
    class="c0"
    color="#5A8DED"
  >
    Advocacia
  </p>
</div>
`;

exports[`CategoryBadge renders enfermagem case correctly 1`] = `
.c0 {
  margin-right: 1em;
  color: #fff;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  max-width: 70%;
  font-size: 1.4em;
  font-weight: 600;
  padding: 0.2em 2em;
  border-radius: 1.5em;
  line-height: 1.8em;
  background-color: #F778C1;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 1.5em;
}

<div>
  <p
    class="c0"
    color="#F778C1"
  >
    Enfermagem
  </p>
</div>
`;

exports[`CategoryBadge renders fisioterapia case correctly 1`] = `
.c0 {
  margin-right: 1em;
  color: #fff;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  max-width: 70%;
  font-size: 1.4em;
  font-weight: 600;
  padding: 0.2em 2em;
  border-radius: 1.5em;
  line-height: 1.8em;
  background-color: #ED985A;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 1.5em;
}

<div>
  <p
    class="c0"
    color="#ED985A"
  >
    Fisioterapia
  </p>
</div>
`;

exports[`CategoryBadge renders fonoaudiologia case correctly 1`] = `
.c0 {
  margin-right: 1em;
  color: #fff;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  max-width: 70%;
  font-size: 1.4em;
  font-weight: 600;
  padding: 0.2em 2em;
  border-radius: 1.5em;
  line-height: 1.8em;
  background-color: #7C65DE;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 1.5em;
}

<div>
  <p
    class="c0"
    color="#7C65DE"
  >
    Fonoaudiologia
  </p>
</div>
`;

exports[`CategoryBadge renders medicine correctly 1`] = `
.c0 {
  margin-right: 1em;
  color: #fff;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  max-width: 70%;
  font-size: 1.4em;
  font-weight: 600;
  padding: 0.2em 2em;
  border-radius: 1.5em;
  line-height: 1.8em;
  background-color: #2AC390;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 1.5em;
}

<div>
  <p
    class="c0"
    color="#2AC390"
  >
    Medicina
  </p>
</div>
`;

exports[`CategoryBadge renders nutrição case correctly 1`] = `
.c0 {
  margin-right: 1em;
  color: #fff;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  max-width: 70%;
  font-size: 1.4em;
  font-weight: 600;
  padding: 0.2em 2em;
  border-radius: 1.5em;
  line-height: 1.8em;
  background-color: #A0C32A;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 1.5em;
}

<div>
  <p
    class="c0"
    color="#A0C32A"
  >
    Nutrição
  </p>
</div>
`;

exports[`CategoryBadge renders odontologia case correctly 1`] = `
.c0 {
  margin-right: 1em;
  color: #fff;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  max-width: 70%;
  font-size: 1.4em;
  font-weight: 600;
  padding: 0.2em 2em;
  border-radius: 1.5em;
  line-height: 1.8em;
  background-color: #51CECE;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 1.5em;
}

<div>
  <p
    class="c0"
    color="#51CECE"
  >
    Odontologia
  </p>
</div>
`;

exports[`CategoryBadge renders pessoa cuidadora correctly 1`] = `
.c0 {
  margin-right: 1em;
  color: #fff;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  max-width: 70%;
  font-size: 1.4em;
  font-weight: 600;
  padding: 0.2em 2em;
  border-radius: 1.5em;
  line-height: 1.8em;
  background-color: #5A8DED;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 1.5em;
}

<div>
  <p
    class="c0"
    color="#5A8DED"
  >
    Pessoa Cuidadora
  </p>
</div>
`;

exports[`CategoryBadge renders psicologia case correctly 1`] = `
.c0 {
  margin-right: 1em;
  color: #fff;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  max-width: 70%;
  font-size: 1.4em;
  font-weight: 600;
  padding: 0.2em 2em;
  border-radius: 1.5em;
  line-height: 1.8em;
  background-color: #A97FE2;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 1.5em;
}

<div>
  <p
    class="c0"
    color="#A97FE2"
  >
    Psicologia
  </p>
</div>
`;
